sections/ai_rules.tex

```diff
@@ -69,12 +69,14 @@ \subsection*{Combat against AI}
 These rules apply during Combat in \textbf{Solo}\index{Solo Mode} and \textbf{Cooperative} Scenarios.
 When Neutral enemies or AI Heroes activate a unit, they follow a set of automatic instructions:\par
 
-Enemy Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units prioritize attacking Units of the same tier.
-If this is impossible, they attack the closest Unit, prioritizing the lowest tier one.\par
-Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units prioritize attacking other Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units of the same tier, then lower tier, and finally higher tier.
-If there are no Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units for them to target, they prioritize Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units in the same tier order.
-If there's more than one valid target, they attack the closest one.\par
-If there's ever a tie between equally valid targets for any Units, the player chooses which Unit is attacked.
+Enemy Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units prioritize attacking Units of the \textbf{same} tier.
+If this is impossible, they attack the Unit of a lower tier (in tier \textbf{descending} order, down to bronze), and if that is also impossible, they attack the Unit of a higher tier (in tier \textbf{ascending} order).\par
```

Comment:
How is this sentence improving the existing text? It doubles the sentence length, making it harder to understand. Can you come up with an example where it actually helps?

Reply:
Do you mean that gold shooter prioritizes silver target over a bronze one?

Reply:
Precisely (as long as both these targets are ranged, or if there are no ranged targets at all)

For gold shooter the order is: gold ranged, silver ranged, bronze ranged, azure ranged, gold non-ranged, silver non-ranged, bronze non-ranged, azure non-ranged.

Yes, it makes the text longer, but this clarification was missing.

See todays discussion in discord

Reply:
Is It possible to add that as an example with \textit {}
For gold shooter the order is: gold ranged, silver ranged, bronze ranged, azure ranged, gold non-ranged, silver non-ranged, bronze non-ranged, azure non-ranged.

Reply:
different symbol meaning?

Reply:
I mean the colored stars and the symbol for archer, ground and flying units. Sorry for the bad english.